hail/src/main/scala/is/hail/backend/spark/SparkBackend.scala

```diff
@@ -519,18 +531,25 @@ class SparkBackend(
           case Right((pt, addr)) => GetFieldByIdx(EncodedLiteral.fromPTypeAndAddress(pt, addr, ctx), 0)
         }
         log.info(s"finished execution of query $queryID")
-        literalIR
+        val id = UUID4().id
```

Comment:
UUIDs are kinda big and weird. Can we use a random long? Or an incrementing index?